xbmc/guilib/guiinfo/MusicGUIInfo.cpp

```diff
@@ -313,6 +313,36 @@ bool CMusicGUIInfo::GetLabel(std::string& value, const CFileItem *item, int cont
         return true;
       }
       break;
+      case LISTITEM_BITRATE:
+      {
+        int BitRate = tag->GetBitRate();
+        if (BitRate > 0)
+        {
+          value = StringUtils::Format("%i", BitRate);
+          return true;
+        }
+        break;
+      }
+      case LISTITEM_SAMPLERATE:
+      {
+        int sampleRate = tag->GetSampleRate();
+        if (sampleRate > 0)
+        {
+          value = StringUtils::Format("%i", sampleRate);
```

Comment:
Why display it as a float ?  I did this originally when I had sort routines for displaying label2.  I even had tests for whether or not there was a decimal point (44.1Khz vs 96Khz) and dropping the .0 decimal because there was no need to display it.  But having removed all that, I see no need to display it as a float.  If I was formatting it then yeah, I'd do some processing on it and make it pretty but I'm not so I think an int is fine. 44100, 96000, 192000 vs 44.1, 96, 192.  Second is neater but what does a skinner want to see to be able to show a .png flag instead of the value ?  Originally I formatted it up for label 2 as the second and added Khz to the label which looked good but probably isn't what is wanted in a listitem.

Reply:
eg, this is what I did previously

```
float sampleRate = static_cast<float>(music->GetSampleRate());
       if (static_cast<int>(sampleRate) % 1000 == 0) // drop .x if x == 0 (44.1 Khz vs 96 Khz)
        value = StringUtils::Format("%d Khz", static_cast<int>(sampleRate/1000));
      else
        value = StringUtils::Format("%.1f Khz",(sampleRate/1000.0));
```

Reply:
Personally I prefer the formatted version where the listitem would be `44.1Khz` but I don't know if Khz needs localizing (it's not in strings.po on its own, just as part of another string #458).  I could even do the divide anyway and set it to 44.1 and the skin could supply the Khz part. 
Estuary appears to use a formatted value and adds Khz /kpbs itself so I guess a slightly formatted value of 44.1, 96 etc would be ok?  See https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/xml/DialogSeekBar.xml#L91L136

Reply:
Info label data returned from core should be consistent what skinners then do with it is up to them. So  core should return LISTITEM_SAMPLERATE in the same number format as the existing _comparable_ player info label MUSICPLAYER_SAMPLERATE.  Leave fancy formatting to the skin rather than hard code in core.